api/challenge_api/Gemfile.lock

```diff
@@ -0,0 +1,181 @@
+GEM
```

Comment:
Commit the updated version of this file after making edits to the Gemfile above.